doc/user_manual/postprocessor.tex

```diff
@@ -1617,6 +1616,56 @@ \subsubsection{RavenOutput}
 \end{lstlisting}
 The resulting PointSet has \emph{time} as an input and \emph{first} as an output.
 
+%%%%%%%%%%%%%% ETimporter PP %%%%%%%%%%%%%%%%%%%
+
+\subsubsection{ETimporter}
```

Comment:
Should we use ```ETImporter```?

Reply:
If you are referring to the case change, then I would agree with @wangcj05